src/styles/base.css

```diff
@@ -13,6 +13,28 @@ html {
   --sl-line-height: 1.5;
   --sl-text-code: var(--sl-text-base);
   --sl-text-code-sm: var(--sl-text-sm);
+  --sl-text-sm: 12px;
+  --sl-text-2xs: 12px;
+  --sl-text-xs: 12px;
+  --sl-text-sm: 12px;
+  --sl-text-base: 12px;
+  --sl-text-lg: 12px;
+  --sl-text-xl: 12px;
+  --sl-text-2xl: 12px;
+  --sl-text-3xl: 12px;
+  --sl-text-4xl: 12px;
+  --sl-text-5xl: 12px;
+  --sl-text-6xl: 12px;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: 12px !important;
```

Comment:
I do find visual hierarchy of headings helpful on docs pages, especially longer ones. But I'm fine with trying this for now and revisiting later.

Reply:
Ok yeah you guys are right the hierarchy is lost. I made some changes in #61. Let me know if that improves it.